tests/fork/config-none.json

```diff
@@ -0,0 +1,17 @@
+{
+    // Mystikos configuration version number
+    "version": "0.1",
+
+    // OpenEnclave specific values
+    "Debug": 1,
+    "StackMemSize": "256k",
+    "NumUserThreads": 2,
+    "ProductID": 1,
+    "SecurityVersion": 1,
+
+    // Mystikos specific values
+    "MemorySize": "40m",
+    "ApplicationPath": "/bin/hello",
+    "HostApplicationParameters": true,
+    "ForkMode":"none"
```

Comment:
Is this the default if the option is omitted?

Reply:
"ForkMode":"none"  = myst_fork_none  = 0
The config structure is either calloc-ed or malloc-ed/memset-ed